packages/glsp-playwright-core/src/debugger/extractor.ts

```diff
@@ -0,0 +1,35 @@
+import { Locator } from '@playwright/test';
+import { SVGMetadata } from '~/glsp/graph/svg-metadata-api';
```

Comment:
Is there a reason you are using alias imports here? I'm just curious cause I never used them before so I wonder what the advantage as we end of using both relative and alias paths anyway.

Reply:
In this particular case, there would not be any benefit. However in general, I dont like using `../` multiple times, I think that makes it harder to read.

Reply:
Since it is only a compile-time feature anyway, I'm fine with that. I just want to make sure that we have some consistency here. So currently "the rule" seems to be that we use `../` at most two times (e.g., `interactable.ts` or `marker.po.ts`) and then use the alias.